### PR TITLE
7.0 metafiles

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[![Coverage Status](https://coveralls.io/repos/OCA/account-financial-reporting/badge.png?branch=7.0)](https://coveralls.io/r/OCA/account-financial-reporting?branch=7.0)
+
+OpenERP account financial reports
+=================================
+
+This project aims to deal with modules related to financial reports. You'll 
+find modules that print legal and official reports. This includes, among 
+others:
+
+* One module based on webkit and totally rewritten by camptocamp, for standard
```
Comment (rvalyi) [inline README.md:11]: minor typo: camptocamp -> Camptocamp

**Comment by rvalyi** [comment]
:+1:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+env:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (laetitia-gangloff) [inline .travis.yml:15]: To make travis happy you can add these install lines:
- git clone https://github.com/OCA/reporting-engine.git ${HOME}/addons-reporting-engine -b 7.0
- sudo wget http://downloads.sourceforge.net/project/wkhtmltopdf/0.12.1/wkhtmltox-0.12.1_linux-precise-amd64.deb -P /tmp/
- sudo dpkg -i /tmp/wkhtmltox-0.12.1_linux-precise-amd64.deb

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="7.0" ODOO_REPO="odoo/odoo"
+  - VERSION="7.0" ODOO_REPO="OCA/OCB"
```
Comment (laetitia-gangloff) [inline .travis.yml:8]: To make travis happy:
- VERSION="7.0" ODOO_REPO="odoo/odoo" EXCLUDE="report_xls,base_report_assembler"
- VERSION="7.0" ODOO_REPO="OCA/OCB" EXCLUDE="report_xls,base_report_assembler"

Comment (pedrobaeza) [inline .travis.yml:8]: I have included them as dependency on 2533a4a. Let's wait for new Travis result

**Comment by pedrobaeza** [comment]
Now it's OK about report_xls, but there is another error for one test. This is for another PR.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+virtualenv:
+  system_site_packages: true
+
+install:
+  - git clone https://github.com/OCA/reporting-engine ${HOME}/reporting-engine -b ${VERSION}
+  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
```
Comment (laetitia-gangloff) [inline .travis.yml:15]: I think, you also need to install wkhtmltopdf:
- sudo wget http://downloads.sourceforge.net/project/wkhtmltopdf/0.12.1/wkhtmltox-0.12.1_linux-precise-amd64.deb -P /tmp/
- sudo dpkg -i /tmp/wkhtmltox-0.12.1_linux-precise-amd64.deb

Comment (pedrobaeza) [inline .travis.yml:15]: Not for now, because yaml tests only check methods and results like variables, but doesn't render HTML nor convert to PDF.

Comment (laetitia-gangloff) [inline .travis.yml:15]: But it is what the error message says lines 3151:
https://travis-ci.org/OCA/account-financial-reporting/jobs/31817934#L3151

except_osv: (u'Wkhtmltopdf library path is not set', u'Please install executable on your system (sudo apt-get install wkhtmltopdf) or download it from here: http://code.google.com/p/wkhtmltopdf/downloads/list and set the path in the ir.config_parameter with the webkit_path key.Minimal version is 0.9.9')

Comment (pedrobaeza) [inline .travis.yml:15]: OK, I see, I was looking at odoo/odoo test, and this is the message:

```
2014-08-06 14:56:07,690 23135 ERROR openerp_test openerp.tools.yaml_import: (u'No period found', '')
```

But in OCB this is fixed. I remember that this was a base problem. Maybe for odoo/odoo test we must exclude the conflictive module.

I'm fixing both things. Thanks for pointing out.

**Comment by laetitia-gangloff** [comment]
It seems the rest is pep8 issue, so it is ok for me :) :+1: 
Thank you !

**Pull request merged**